src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
```

Comment:
Try not to leave empty spaces in opened rule declarations so code would be smaller and easier to read/work with.